src/typescript-worker/typescript-builder.ts

```diff
@@ -10,6 +10,8 @@ import {PackageJson} from './util.js';
 import {makeLspDiagnostic} from './diagnostic.js';
 import {WorkerContext} from './worker-context.js';
 
+const PROCESSED_FILE_ENDINGS = ['.ts', '.jsx', '.tsx'];
```

Comment:
```suggestion
const COMPILED_FILE_ENDINGS = ['.ts', '.jsx', '.tsx'];
```

(Just a nit, since `.js` is "processed", it's just not compiled)